src/get-started/test-drive/index.md

```diff
@@ -10,11 +10,24 @@ next:
 toc: false
 ---
 
-This page describes how to create a new Flutter app from templates, run it,
-and experience "hot reload" after you make changes to the app.
+This page describes the following procedures:
 
-Select your development tool of choice for writing, building, and running
-Flutter apps.
+1. How to create a new Flutter app from templates.
+1. How to run the created Flutter app.
+1. How to use "hot reload" after you make changes to the app.
+
+Details for these procedures vary according to the IDE you use.
+
+The first two options listed rely on the Flutter plugin for
+the respective IDE.
+Flutter supports language plugins for Visual Studio Code,
+Android Studio, and IntelliJ IDEA Community, Educational, and
+Ultimate editions.
```

Comment:
It's not that Flutter supports plugins but rather those editors have Flutter plugins.